Packaging/fedora/devilutionx.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=devilutionX
+GenericName=DevilutionX
+Comment=Play Diablo I on Linux
+Exec=/var/games/devilutionx/devilutionx
+Icon=devilutionx
```

Comment:
Possibly you could use one of those icons in the Mac folder, there is a 1024x1024px version and some all the way down to 16x16px